templates/portal/teaching.html

```diff
@@ -0,0 +1,101 @@
+            Should any issues arise though, we will work with you to resolve them as quickly as possible.</p>
+          <h3>How do I request to use SciLifeLab Serve for teaching?</h3>
+          <p>To ensure that the functionality intended for teaching is not used for other purposes, and that only one
+            course is running at any given time, we have established a specific <a href="#application">application
+              form</a> for access to this functionality. The <a href="#application">form</a>, found below, enables you
+            to provide details about yourself and your planned course/workshop. Upon approval of your application,
+            your SciLifeLab Serve account will be granted permission to create multiple instances of JupyterLab
+            notebook and/or RStudio. Each instance that you create will be assigned a unique URL that you can assign
+            to a given student/attendee.</p>
+          <h3>Who is eligible to apply?</h3>
+          <p>We accept applications for courses, workshops, and webinars that are organised by Swedish universities or
+            SciLifeLab infrastructure units. The courses/workshops/webinars should be aimed life science researchers
+            at the doctoral level or above.</p>
+          <h3>What kind of compute and storage capacity will we receive? Is there a limit to participant number?</h3>
+          <p>We will discuss your individual requirements for CPUs and memory with you. We expect to be able to cover
+            the needs of most courses. For reference, SciLifeLab Serve has previously been used to teach courses
+            involving up to 30 paticipants without issue.</p>
+          <h3>What support will be provided?</h3>
+          <p>Before the course, we will be available to answer any questions that you might have regarding features of
+            SciLifeLab Serve, setting up the required instances for students, and the interface of SciLifeLab Serve.
+            During the course, we will be on hand to answer any questions as quickly as possible and will monitor our
+            servers throughout. In the event of a server issue, we will do all we can to resolve it quickly. We
+            recommend having a backup plan in case there are unforeseen issues that cannot be resolved promptly.</p>
+          <h3>How is SciLifeLab Serve funded?</h3>
+          <p>SciLifeLab Serve is supported by a grant from the Knut and Alice Wallenberg Foundation given to
+            SciLifeLab for the purpose of supporting research in data-driven life science in Sweden. SciLifeLab Serve
+            is available free of charge to researchers affiliated with Swedish research institutions, regardless of an
+            affiliation with SciLifeLab. It is also offered to ScLifeLab infrastructure units. Refer to our <a
+              href="{% url 'portal:about' %}">'about' page</a> for more information.</p>
+
+        <h2 id="application">Application form</h2>
```

Comment:
This headline has different style. Looks a bit off, but fixing style is maybe not within the scope of this PR.

Reply:
Agree, and yes.